packages/wallet/api/test/test-lib-wallet.js

```diff
@@ -1205,7 +1205,7 @@ test('lib-wallet performAction suggestIssuer', async t => {
     type: 'suggestIssuer',
     data: { petname: 'bucksIssuer', boardId: bucksIssuerBoardId },
   });
-  const done = await wallet.performAction({ action });
+  const done = await wallet.performAction({ spendAction: action });
```

Comment:
suggestIssuer is a `spendAction`? I suppose that's correct but just a sort of de-optimization.

Reply:
I haven't tried getting a non-spend action end-to-end working, I assumed that was still needing design but perhaps this can be the first one we try with. We can revisit.